HACKING.md

```diff
@@ -160,6 +160,38 @@ your charm to a controller using that version of Juju. For example, with microk8
 We rely on automation to [update charm pins](.github/actions/update-charm-pins/) of
 a bunch of charms that use the operator framework. The script can be run locally too.
 
+# Contributing
+
+Changes are proposed as [pull requests on GitHub](https://github.com/canonical/operator/pulls).
+
+Pull requests should have a short title that follows the
+[conventional commit style](https://www.conventionalcommits.org/en/) using one of these types:
+
+* chore
+* ci
+* docs
+* feat
+* fix
+* perf
+* refactor
+* revert
+* test
+
+If the PR is limited to one of the following scopes, also include the scope in the title,
+and otherwise leave off the scope.
+
+* harness
+
```

Comment:
Was there perhaps an intention to add more scope examples in this doc?
Otherwise "one of the following scopes" reads odd.

Reply:
The intention (if I recall the discussion correctly) was that we would potentially add more scopes in the future, as we find something that would be useful. So I wrote it to easily accommodate that. I could rewrite to just focus specifically on `(harness)` for now if that would be better.

Reply:
Yeah, I wondered that too. I kind of agree with Dima -- would probably be better to have it read well for what it is now (just one scope) and update it when we add more. Or note explicitly that "at present there's only one" or something.